src/main/java/org/broadinstitute/consent/http/models/UserFile.java

```diff
@@ -0,0 +1,136 @@
+package org.broadinstitute.consent.http.models;
+
+import org.broadinstitute.consent.http.enumeration.UserFileCategory;
+
+import java.io.InputStream;
+import java.util.Date;
+import java.util.Objects;
+
+public class UserFile {
```

Comment:
Nit, naming (naming is hard!). These are not necessarily `User` `Files`. Files could exist for any conceivable entity. Could we name this something more generic such as `FileObject` or `FileStorageObject` or anything in that genre.

Reply:
good point, I was trying to avoid just `File` because I felt a class with the name `File` has a specific meaning. `FileStorageObject` seems apt.